addons/web_editor/static/src/scss/wysiwyg.scss

```diff
@@ -174,6 +174,11 @@ $o-we-toolbar-color-clickable-active: $o-we-bg-darkest;
         justify-content: center;
     }
 
+    .btn.fa {
+        width: 32px;
+        box-sizing: border-box;
+    }
```

Comment:
If this is a fix for mass_mailing only, aren't you concerned you're gonna break the layout in other instances of wysiwyg?

Reply:
This is only for instance of .btn.fa of the editor toolbar. So there is not much instances of those and I've checked that it does not break the layout.

Reply:
But is this actually bounded by the toolbar? Wouldn't CSS on global .btn.fa apply everywhere as long as that file is loaded? If it concerns only the toolbar, I'd be more specific, like `.oe_toolbar .btn.fa` otherwise you might not have encountered a case this broke but that doesn't mean it doesn't exist. That's what I meant with my comment.

Reply:
It _is_ bounded by the toolbar. I'm marking this as resolved.

Reply:
After a doubt and an investigation, I found a better fix that could cover more issues.
The problem was caused by a rule made for mail clients that reached the toolbar btn (and could reach other part).

In the template `mass_mailing_mail_style`:
```css
* {
    box-sizing: border-box !important;
}
```

Reply:
> It _is_ bounded by the toolbar. I'm marking this as resolved.

jfc my bad that will teach me to look at github on my phone on my off day :eyes: On the phone I can't see the context and it looked like it wasn't. Apologies.